code/hispania/game/objects/items/weapons/storage/erp.dm

```diff
@@ -0,0 +1,89 @@
+/obj/item/storage/toolbox/mre //godoforeos: Jason Conrad
+	name = "emergency ration pack"
+	desc = "Silvery plastic package, with the letters \"ERP\" pasted onto the front. Seems air tight, and vacuumed sealed. \
+	The packaging holds usage information within the fineprint: \
+	\"Instructions: Remove contents from packaging, open both mre container and ration can. \
+	Enriched crackers and morale bar contains medicinal additives for field performace, DO NOT OVERCONSUME.\""
+	icon = 'icons/hispania/obj/items.dmi'
+	icon_state = "erp_closed"
+	item_state = "erp"
+	force = 5
+	throwforce = 5
+	w_class = WEIGHT_CLASS_NORMAL
+	max_combined_w_class = 0
+	storage_slots = 5
+	hispania_icon = TRUE
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	var/openmre = FALSE
+
+/obj/item/storage/toolbox/mre/New()
+	..()
+	new /obj/item/storage/toolbox/comida_mre(src)
+	new /obj/item/storage/toolbox/lata_arepa(src)
+	new /obj/item/storage/fancy/mre_cracker(src)
+	new /obj/item/reagent_containers/food/snacks/choco_mre(src)
+	new /obj/item/kitchen/utensil/fork(src)
+
+/obj/item/storage/toolbox/mre/show_to(mob/user)
+	if(!openmre)
+		to_chat(user, "<span class='warning'>You tear \the [src] open.")
+		icon_state = "erp_open"
+		openmre = TRUE
+		playsound(src, 'sound/items/poster_ripped.ogg', 50, 1)
+	..()
+
+/obj/item/storage/toolbox/lata_arepa
+	name = "emergency arepa can"
+	desc = "A silver can that contains a random arepa, made for special emergencies."
+	icon_state = "ration_can"
+	item_state = "erp"
+	force = 1
+	throwforce = 1
+	w_class = WEIGHT_CLASS_NORMAL
+	max_combined_w_class = 0
+	storage_slots = 1
+	hispania_icon = TRUE
+	var/openmre = FALSE
+
+/obj/item/storage/toolbox/lata_arepa/New()
+	..()
+	var/newarepa = pick(/obj/item/reagent_containers/food/snacks/arepa_ham_cheese, /obj/item/reagent_containers/food/snacks/arepa_salmon, /obj/item/reagent_containers/food/snacks/fruit_arepa, /obj/item/reagent_containers/food/snacks/cheese_arepa, /obj/item/reagent_containers/food/snacks/sweet_arepa, /obj/item/reagent_containers/food/snacks/ghost_arepa, /obj/item/reagent_containers/food/snacks/arepa_ham_cheese, /obj/item/reagent_containers/food/snacks/arepa_ham, /obj/item/reagent_containers/food/snacks/arepa_cheese)
+	new newarepa(src)
+
+
+/obj/item/storage/toolbox/lata_arepa/remove_from_storage(obj/item/I, atom/new_location)
```

Comment:
Una vez la abres entiendo y tomas el contenido entiendo que no puedas volver a insertarlo dentro, pero cuando intentas insertarla dentro te marca que se encuentra lleno el storage. 

![image](https://user-images.githubusercontent.com/46639834/118017527-59ae8d80-b31c-11eb-862f-e92b5ee2d3c9.png)

"The emergency ration is full, make some space" 

Ya sea que quieras cambiarlo para que puedas volver a meter la comida o que el objeto cambie a tipo trash pero no puede mostrar que se encuentra lleno cuando no hay nada, actualmente lo mas similar es el sistema en snacks.dm para Trash.

Reply:
Esto aplica tanto para las bolsas dentro del ERP como para el ERP mismo.

Reply:
Si hay alguna duda de como poder resolverlo me pingeas por Discord o me mandas DM. Cualquier cosa en cuanto quede resuelto el detalle de que es un objeto heredado de toolbox y el hecho de que marcan las bolsas estar llenas cuando no tienen nada esta listo para mergear.